### ISSUE TITLE
user, password and dbname from settings.json is ignored

### ISSUE THREAD

**Issue by janhapke** [Microsoft/vscode-postgresql#21]
- PGSQL Extension Version: 0.1.0
- VSCode Version: Version 1.32.3 (1.32.3)
- OS Version: macOS High Sierra 10.13.6 (17G5019)

Steps to Reproduce:

1. settings.json
```
{
    "window.zoomLevel": -1,
    "pgsql.connections": [
        {
            "host": "localhost",
            "port": "54321",
            "dbname": "my_db",
            "user": "postgres",
            "password": "password",
            "emptyPasswordInput": false,
            "profileName": "My DB"
        }
    ]
}
```
2. Command Palette > PostgreSQL: New Query
3. Select Connection Profile

I am asked to enter the username and then the password (although both is specified in settings.json)

When I try to run a query, I get an error saying that `relation "mytable" does not exist`.

I try to list the tables and it appears I am not connected to the correct database (even though the name is specified in settings.json as well).

**Comment by akamath46297** [comment]
@janhapke this is now fixed for Dbname and user name with [Release 0.2.0](https://github.com/Microsoft/vscode-postgresql/releases/tag/v0.2.0)

For security purposes we have not handled this for password. We are looking into ways where we can store the password in some cred store so that it is secure

**Comment by akamath46297** [comment]
Opened a separate enhancement for handling password #35
